README.md

```diff
@@ -1,3 +1,39 @@
 # SnowMan
 
-This is a console Swift application. It plays like Hangman but without the gore.
+Snowman is a Swift console application. The object of the game is to guess the given word or phrase before the snowman "melts". With each incorrect guess, the snowman will lose one of its snowman features until he is but a puddle. 
+
+## Requirements
+
+[Swift](https://swift.org/) 
+[Swift Package Manager](https://swift.org/package-manager/)
+
+## Gameplay
+
+To play the game... TBD
+
+## Running the Application
+
+SnowMan was built using the Apple Platform [Swift Development Snapshot](https://swift.org/download/#latest-development-snapshots) from March 1, 2016 for Apple Swift version 2.2.
+
+To compile the application for use, type 
+```swift
+swift build
+```
+
+Then run the `main.swift` file within the `Application` branch of the `Sources` file. 
```

Comment:
"Technically" you're not running the `main.swift` file, you're just running the application.